test/unit-tests/core-index-stream.test.js

```diff
@@ -7,7 +7,7 @@ const {
   create,
   replicate,
   generateFixture,
-  throttledIdle,
+  throttledDrain: throttledIdle,
```

Comment:
did you mean to rename references to throttledIdle to throttledDrain for this file? (i've run into this issue occasionally when using vscode's reference renaming feature 😄 )